Lib/pdb.py

```diff
@@ -1607,6 +1608,11 @@ def do_run(self, arg):
         sys.argv.  History, breakpoints, actions and debugger options
         are preserved.  "restart" is an alias for "run".
         """
+        if self.mode == 'inline':
+            self.error('run/restart command is not allowed with inline breakpoints.\n'
+                       'Use the command line interface if you want to restart your program\n'
+                       'e.g. "python -m pdb myscript.py"')
```

Comment:
We can, but we need to print something when the user uses `restart`. If we "remove" the command, it would be like executing `restart` in REPL - which would probably result in a `NameError`. Don't you think that's even more confusing for the users?

Reply:
Wy name error? Isn't there a standard message when you use a non-existing command?
If they get confused there is the documentation.

Reply:
When user types random things in pdb, they will be checked against the existing commands first, then parsed as a statement/evaluation. So when users type `x = 1` it will be executed, instead of showing an error like "command x does not exist".